R/admiraldev-package.R

```diff
@@ -2,8 +2,7 @@
 #' @importFrom dplyr across arrange bind_rows case_when desc ends_with filter
 #'   full_join group_by if_else mutate n pull rename row_number select slice
 #'   starts_with transmute ungroup n_distinct union distinct summarise coalesce
-#'   bind_cols na_if tibble
-#' @importFrom magrittr %>%
```

Comment:
Could this break the package if `dplyr` is no longer exporting `%>%`? The reexports in `dplyr` are hidden in the documentation. You can find them via search but they are not mentioned on the "Reference" page. Thus they may be removed without deprecation?

What is the benefit of importing the reexport instead of the original function?